packages/components/src/components/common/y-axis/as-y-axis.scss

```diff
@@ -1,15 +1,16 @@
 .y-axis {
-  --widget-axis-line-color: var(--as--color--type-02, #747474);
+  --widget-axis-text-color: var(--as--color--type-01, #{$color-type-01});
+  --widget-axis-line-color: rgba(250, 34, 34, 1);
```

Comment:
I'm OK with this because it would let us change the line color for some themes but there's no need to 'link' to another airship variable, but I just wanted to point it out in case you had another idea and forgot to change it. It's the only case left with an rgba literal.